alerta-covid/app/controllers/areas_controller.rb

```diff
@@ -0,0 +1,69 @@
+class AreasController < ApplicationController
+  before_action :set_area, only: %i[ show edit update destroy ]
+
+  # GET /areas or /areas.json
+  def index
+    @areas = Area.all
+  end
+
+  # GET /areas/1 or /areas/1.json
+  def show
+  end
+
+  # GET /areas/new
+  def new
+    @area = Area.new
+  end
+
+  # GET /areas/1/edit
+  def edit
+  end
+
+  # POST /areas or /areas.json
+  def create
+    @area = Area.new(area_params)
+
+    respond_to do |format|
+      if @area.save
+        format.html { redirect_to @area, notice: "Area was successfully created." }
+        format.json { render :show, status: :created, location: @area }
+      else
+        format.html { render :new, status: :unprocessable_entity }
+        format.json { render json: @area.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # PATCH/PUT /areas/1 or /areas/1.json
+  def update
+    respond_to do |format|
+      if @area.update(area_params)
+        format.html { redirect_to @area, notice: "Area was successfully updated." }
+        format.json { render :show, status: :ok, location: @area }
+      else
+        format.html { render :edit, status: :unprocessable_entity }
+        format.json { render json: @area.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # DELETE /areas/1 or /areas/1.json
+  def destroy
+    @area.destroy
+    respond_to do |format|
```

Comment:
you dont need to respond to Json here, sice we have the api area controller

Reply:
Removed json responses for web module controller